Client/src/Components/product-reviews/RecommendProduct.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import css from "./styles.css";
+
+// RecommendProduct takes in reccommend which is a boolean true/false.
+// If true, it will display a message saying the reviewer recommends the product.
+// If false, it will return false.
+function RecommendProduct({ recommend }) {
+  return (
+    <div>
+      {recommend && (
```

Comment:
if recommend is a boolean, use the naming convention of starting the name with 'has' or 'is'. For example: 'isRecommended'.